main.go

```diff
@@ -1086,6 +1086,27 @@ func sidekickMain(ctx *cli.Context) {
 			ClientSessionCache:       tls.NewLRUClientSessionCache(tlsClientSessionCacheSize),
 		}
 		server.TLSConfig = tlsConfig
+	} else if ctx.String("auto-tls-host") != "" {
+		cert, key, err := generateTLSCertKey(ctx.String("auto-tls-host"))
+		if err != nil {
+			console.Fatalln(err)
+		}
+		console.Printf("Generated TLS certificate for host '%s'\n", ctx.String("auto-tls-host"))
+		console.Printf("certFile:\n%s\n", string(cert))
+		console.Printf("keyFile:\n%s\n", string(key))
+		certificates, err := tls.X509KeyPair(cert, key)
+		if err != nil {
+			console.Fatalln(err)
+		}
```

Comment:
we only need to print public.crt not private.key and also make it available for someone to download it